frontend/src/components/Tree/Tree.jsx

```diff
@@ -95,12 +96,14 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick, onRightClick) {
     .attr(
       "transform",
       (d) => `
-        rotate(${(d.x * 180) / Math.PI - 90})
+        rotate(${(d.x * TreeConf.NODE_ROTATE_FACTOR) / Math.PI - 90})
         translate(${d.y},0)
       `
     )
-    .attr("fill", (d) => (d.data.numChildren === 0 ? "#b30000" : "#555"))
-    .attr("r", 6)
+    .attr("fill", (d) =>
+      d.data.numChildren === 0 ? TreeConf.LEAF_COLOUR : TreeConf.NODE_COLOUR
```

Comment:
Just leaving this here as a way to remind you to change the LEAF_COLOR and NODE_COLOR!

Reply:
Done! Lmk if any issues with the colours